app/scripts/controllers/permissions/permissionsMethodMiddleware.js

```diff
@@ -4,7 +4,7 @@ import { ethErrors } from 'eth-json-rpc-errors'
 /**
  * Create middleware for handling certain methods and preprocessing permissions requests.
  */
-export default function createMethodMiddleware ({
+export default function createPermissionsMethodMiddleware ({
```

Comment:
This file and all references to its export were renamed to disambiguate it from the new top-level `methodMiddleware`.